portfolio/src/main/webapp/style.css

```diff
@@ -13,4 +13,9 @@
   margin-right: auto;
   width: 650px;
   line-height:50px;
+}
+
+#vert-image {
+    width: 225px;
```

Comment:
nit: https://google.github.io/styleguide/htmlcssguide.html#Indentation
Not super important for this project but it will be helpful when you are all working on the same project.